### DISCUSSION TITLE
Support for bean validation `TYPE_USE`

### DISCUSSION THREAD

**Discussion by rjpipino** [instancio/instancio#910] [Q&A]
Test Class

```java
public class Foo {
  private List<@Negative Long> list;

  @Size(min = 1, max = 1)
  private Map<@Email String, @Negative Integer> map;
}
``` 

Generation result:

```
list = {ArrayList@2186}  size = 5
 0 = {Long@2195} 9427
 1 = {Long@2196} 9157
 2 = {Long@2197} 2
 3 = {Long@2198} 7819
 4 = {Long@2199} 8814
map = {HashMap@2187}  size = 1
 "MVDHHY" -> {Integer@2193} 8119
```

Expected all elements in `list` to be negative and `map` key to be valid email format with a negative value.

Is this currently not supported? If not, is there plans to support it? Any possible workarounds?

Thanks

**Comment by armandino** [comment]
Currently it's not supported. I'll add this to the next release. Thanks for the suggestion.

- #911

**Reply by rjpipino** [reply]
Excellent! Thank you!

**Reply by armandino** [reply]
This was released in `4.3.0`. Please give it a try.

**Reply by rjpipino** [reply]
Working great! Thanks for the quick change!
